gammapy/makers/background/reflected.py

```diff
@@ -41,6 +146,25 @@ def run(self, region, center, exclusion_mask=None):
         wcs: `~astropy.wcs.WCS`
             WCS for the determined regions
         """
+        reference_geom = self._create_reference_geometry(region, center)
+        center_pixel = self._get_center_pixel(center, reference_geom)
+
+        region_pix = self._get_region_pixels(region, reference_geom)
+        excluded_pixels = self._get_excluded_pixels(reference_geom, exclusion_mask)
+
+        n_positions = self.n_off_regions + 1
+        increment = FULL_CIRCLE / n_positions
```

Comment:
You could also loop over increments:
```
rotation_angles = np.linspace(0*u.rad, FULL_CIRCLE, n_positions, endpoint=False)
for angle in rotation_angles:
    region_test = region_pix.rotate(center_pixel, angle)
```

Reply:
Your proposal is bugged, since it starts at 0, which is the source position.

Reply:
I think the loop is very clear and easy to reason about.

Reply:
> Your proposal is bugged, since it starts and ends at 0 / 360°, which is the source position.

It misses the `[1:]` indeed, but it does not contain the end point. But it does not matter.